spawn_operator/spawn_operator/lib/spawn_operator/k8s/proxy/service.ex

```diff
@@ -61,7 +61,8 @@ defmodule SpawnOperator.K8s.Proxy.Service do
           "prometheus.io/scrape" => "true"
         },
         "labels" => %{
-          "spawn-eigr.io/controller.version" => "0.5.3"
+          "spawn-eigr.io/controller.version" =>
+            "#{to_string(Application.spec(:spawn_operator, :vsn))}"
```

Comment:
This doesn't work because of InternalVersions the version will always be 0.0.0-local.dev

Reply:
When we build an image we are executing "rewrite_versions" inside the dockerfile, to update it all

Reply:
> When we build an image we are executing "rewrite_versions" inside the dockerfile, to update it all

But vsn will get it from the compiled version of the module which will always be version 0.0.x Do a test to confirm what I'm talking about, but from what I remember this is how it would work

Reply:
@eliasdarruda Forgive me, now I understand what you said. Okay, if we overwrite the dockerfile then it should work accordingly.